docker/worker/Dockerfile

```diff
@@ -88,8 +88,7 @@ COPY --chown=turbinia:turbinia tools/fraken/* /opt/fraken/
 RUN cd /opt/fraken/yara && curl -s https://api.github.com/repos/VirusTotal/Yara/releases/latest | sed -n 's/.*"tarball_url": "\(.*\)",.*/\1/p' | xargs -n1 wget -O - -q | tar -xz --strip-components=1 \
     && ./bootstrap.sh \
     && ./configure \
-    && make && sudo make install && sudo ldconfig \
-    && sudo update-alternatives --install /usr/bin/go go /usr/lib/go-1.16/bin/go 10
```

Comment:
So we don't need to change this "alternative" path?

Reply:
We don't because there'll only be one go version on the system and that'll be 1.18 or whatever the latest go version is, we don't have to hardcode it.